src/routes/notifications/index.test.ts

```diff
@@ -36,6 +36,130 @@ afterEach(() => {
 });
 
 describe('index', () => {
+    describe('#invite-limited', () => {
```

Comment:
Was this accidentally added? or previously deleted?

Reply:
I'm not sure, maybe it was stashed or something. That code is already on the main branch.

Reply:
Hmm, I'm looking at the code and it seems this test was never added for some reason.  I remember looking at this test though.  That's strange, was a test removed at all?